package.xml

```diff
@@ -9,28 +9,26 @@ xsi:schemaLocation="http://pear.php.net/dtd/tasks-1.0 http://pear.php.net/dtd/ta
 hash, integer to string hash and a string to integer hash. The main features
 are low memory consumption and fast serialization and deserialization of
 the sets and hashes.
-  
+
  </description>
  <lead>
   <name>Derick Rethans</name>
   <user>derick</user>
   <email>derick@php.net</email>
   <active>yes</active>
  </lead>
- <date>2012-09-02</date>
- <time>22:44:00</time>
+ <date>2016-05-07</date>
+ <time>17:40:00</time>
  <version>
-  <release>1.0.0</release>
+  <release>2.0.0</release>
```

Comment:
As there are no new features, it probably should just be a 1.1. Going from 1 to 2 implies breaking feature changes.